source/npm/qsharp/ux/circuit-vis/sqore.ts

```diff
@@ -203,10 +206,14 @@ export class Sqore {
     };
 
     const { qubits, componentGrid } = circuit;
-    const { qubitWires, registers, svgHeight } = formatInputs(qubits);
+    const { qubitWires, registers, svgHeight } = formatInputs(
+      qubits,
+      !this.options.isEditable ? this.options.renderLocations : undefined,
```

Comment:
nit: Can this condition be flipped to avoid the negation?
`this.options.isEditable ? undefined : this.options.renderLocations`